rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -11,4 +11,13 @@ if [ -z `getent passwd spinnaker` ]; then
   useradd --gid spinnaker spinnaker -m --home-dir /home/spinnaker
 fi
 
+# install packer
+pushd
+mkdir /packer && \
```

Comment:
Dockerfiles are a bit of a special case, since it's more OK to pollute the `/` directory in a containerized environment. This should end up in `/usr/bin`.